pkg/plugins/common/kustomize/v2/scaffolds/internal/templates/config/kdefault/webhook_manager_patch.go

```diff
@@ -50,33 +50,33 @@ func (f *ManagerWebhookPatch) SetTemplateDefaults() error {
 	return nil
 }
 
-//nolint:lll
+// nolint:lll
+// nolint:lll
```

Comment:
should it be doubled ?

Reply:
No should be only one :-) 
good catcher